content/tutorials/provision-dashboards-and-data-sources.md

```diff
@@ -111,17 +116,19 @@ providers:
     folder: Services # The folder where to place the dashboards
     type: file
     options:
-      path: /var/lib/grafana/dashboards # The path to the dashboard definitions
+      path:  <path to dashboard definitions> # The path to the dashboard definitions
+            # Default path for Windows: C:/Program Files/GrafanaLabs/grafana/public/dashboards # The path to the dashboard definitions
```

Comment:
Looking closer at this, I don't think there's a _default_ path for dashboard definitions. I took the example from https://github.com/grafana/grafana/blob/master/conf/provisioning/dashboards/sample.yaml, but I'm thinking maybe we should ask the user to create a specific directory and set the path themselves here.